python/GangaLHCb/Lib/Applications/GaudiExecUtils.py

```diff
@@ -99,7 +99,6 @@ def prepare_cmake_app(myApp, myVer, myPath='$HOME/cmtuser', myGetpack=None):
     full_path = expandfilename(myPath, True)
     if not path.exists(full_path):
         makedirs(full_path)
-        chdir(full_path)
```

Comment:
Is this line definitely not needed now? Looking at the `getpack` call below, it looks like it's now being called with an absolute path so perhaps it's unnecessary but I wanted to check since it wasn't mentioned in the commit message or the PR description.